src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -33,6 +33,11 @@ public static void Aggregate_AggregateException_Wraps_OperationCanceledException
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, (i, j) => i, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(() => 0, (i, j) => j, (i, j) => i, i => i));
+            Functions.AssertAggregateNotCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate((i, j) => j));
```

Comment:
@stephentoub - also:
I noticed when dealing with these that I wasn't handling "external" cooperative cancellation - say, `Aggregate` throwing an OCE with the appropriate canceled token (since Alternate/NotCanceled are testing "non"-cooperative cancellation).  At least in Aggregate's case the sequential versions don't participate, and wrap it, but the parallel versions **do** participate, which may be surprising.

So, should I:
 1. Ignore, behavior as designed
 2. Raise issue, ignore for now (and/or matrix too large?)
 3. Raise issue, add tests to this PR
 4. Raise issue, add new PR for these tests
 5. ?

Aggregate at least responds appropriately if the underlying enumerable throws the exception, just not if the method itself throws it.